spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/support/AbstractRibbonCommand.java

```diff
@@ -123,9 +123,33 @@ protected static Setter getSetter(final String commandKey, ZuulProperties zuulPr
 	protected static Setter getSetter(final String commandKey,
 			ZuulProperties zuulProperties, IClientConfig config) {
 
-		// @formatter:off
-		Setter commandSetter = Setter.withGroupKey(HystrixCommandGroupKey.Factory.asKey("RibbonCommand"))
-								.andCommandKey(HystrixCommandKey.Factory.asKey(commandKey));
+		RequestContext requestContext = RequestContext.getCurrentContext();
+		String routeId = (String)requestContext.get("proxy");
```

Comment:
There shouldn't be a need for this. The service is available in `IClientConfig config`